packages/ui/src/components/IconResolver/IconResolver.tsx

```diff
@@ -18,7 +18,9 @@ export const IconResolver: FunctionComponent<PropsWithChildren<IconResolverProps
           alt="Kamelet icon"
         />
       );
+    case CatalogKind.Processor:
```

Comment:
@tplevko, considering that we have a single icons list, should we add the `component` condition as well, so both processors and component have their right icons? 

```suggestion
    case CatalogKind.Processor:
    case CatalogKind.Component:
```

Reply:
yes, this makes sense also with the additions to component icons. Updated, thanks!